Activity.js

```diff
@@ -8,7 +8,20 @@ class Activity {
         this.id = id;
     }
 
-    startTimer() {
+    startTimer(minutes, seconds) {
```

Comment:
This looks really great! Very clever use of conditionals here. 

Seems like a good next step would be to add to the else statement some logic to create or show a `log activity` button that would allow the activities to persist on the page.